program/c/src/oracle/upd_aggregate.h

```diff
@@ -131,8 +131,10 @@ static void upd_ema(
 }
 
 static inline void upd_twap(
-    pc_price_t *ptr, int64_t nslots, pc_qset_t *qs )
+    pc_price_t *ptr, int64_t nslots )
```

Comment:
trivial refactor: the qset argument contains a table of constants that only gets used in upd_twap, but was being passed in from upd_aggregate. Moving the table creation into upd_twap made it easier to expose the functions ^ for testing.